proxy/coco/src/peer/run_state.rs

```diff
@@ -104,11 +156,21 @@ pub enum RequestEvent {
         /// The reason the clone failed.
         reason: String,
     },
+    /// Query the network for the `RadUrn`.
+    Queried(RadUrn),
+    /// A urn was requested.
+    Requested(
+        RadUrn,
+        Instant,
+        Option<oneshot::Sender<Option<SomeRequest<Instant>>>>,
```

Comment:
When can this be `None`?

Reply:
We call it a lot when we don't care about the answer, namely in the tests.